source/settings.rst

```diff
@@ -942,6 +942,25 @@ This page allows detailed settings to be modified.
 
     Show tab for GPG information if available.
 
+  .. setting:: Show output history as tab (otherwise as panel)
+
+    The output displayed in the process dialog and the trace output is retained and shown in the output history.
+
+    - With this set, the output history is displayed in a tab in the lower pane of the :ref:`browse-repository` window.
+    - With this unset, the output history is displayed in a panel docked to the lower left corner of the :ref:`browse-repository` window.
+
+      The panel also uses some space of the file status list of the :ref:`browse-repository-tabs-diff` tab.
+
+      The panel height can be adapted using the splitter control of the :ref:`browse-repository-left-panel`.
+
```

Comment:
This paragraph can be removed. I have added a second splitter to the file status list.